rental.rb

```diff
@@ -0,0 +1,10 @@
+# Rental class with data attribute
+class Rental
+  attr_accessor :data, :book, :person
+
+  def initialize(data, book, person)
+    @data = data
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Now after you have made the changes in the add_rentals method in person and book class, you should make `The rental is adding itself to the @book and @person achieving the 'belongs-to' association` so you should have something like this in your initialize `book.rentals << self` and `person.rentals << self`